domain-models-interface-extensions/src/main/java/org/folio/rest/tools/GenerateRunner.java

```diff
@@ -223,6 +224,7 @@ public static void createLookupList(File directory, String name, List<String> su
       paths = pathStream.map(basePath::relativize)
           .filter(path -> pathMatchers.stream()
                             .anyMatch(pathMatcher -> pathMatcher.matches(path)))
+          .sorted()
```

Comment:
WHy does sorting the paths work? Is that necessary? I don't understand why sorting paths would have any effect on those types.

Reply:
The created file currently has random order and with this fix it will always have exactly the same order each time. We want deterministic behaviour.